### PR TITLE
Fixes #11439 - Prevents hammer from reading './config' file

### PR DESCRIPTION
**Pull request by adamruzicka**

### CODE REVIEW

File: bin/hammer [hunk 1]
```diff
@@ -52,12 +52,16 @@ end
 
 CFG_PATH_LOCAL = ['./config/']
 
+HammerCLI::Settings.load_from_paths CFG_PATH_LOCAL
+
 if preparser.config
-  CFG_PATH_LOCAL.unshift preparser.config
+  if File.file? preparser.config
+    HammerCLI::Settings.load_from_file preparser.config
```
Comment (tstrachota) [inline bin/hammer]: Does it work with tilde too? It seems that `load_from_file` does not expand paths.

Comment (adamruzicka) [inline bin/hammer]: The tilde shouldn't even get there. It should get expanded by the shell, unless the user escapes it, in which case I'd think the tilde is part of the path as a literal

Comment (tstrachota) [inline bin/hammer]: That's not completely true. Bash won't expand it when you pass the parameter as a string. E.g.
```
hammer -c '~/some_config.yml'
```
I think this should be supported usage.

**Comment by tstrachota** [comment]
thanks @adamruzicka

**Pull request merged**